include/swift/SIL/SILType.h

```diff
@@ -638,7 +638,10 @@ class SILType {
   /// Canonical way to check if a SILType is move only. Using is/getAs/castTo
   /// will look through moveonly-ness.
   bool isMoveOnlyWrapped() const {
-    return getRawASTType()->is<SILMoveOnlyWrappedType>();
+    // TODO: Is this correct?
```

Comment:
Hmm, we should always be able to get a raw AST type, shouldn't we? I mean your SILType might be `== SILType()` but then the issue lies somewhere else. Seems like this change would pull a `if (silty)` check into this API. I don't think that is right.

Do you remember why this was necessary?

Reply:
That may be a leftover from earlier. I'll see what happens if I remove it. Good catch.